flink-runtime/src/main/java/org/apache/flink/runtime/state/changelog/ChangelogStateBackendHandle.java

```diff
@@ -49,6 +51,9 @@
     List<ChangelogStateHandle> getNonMaterializedStateHandles();
 
     class ChangelogStateBackendHandleImpl implements ChangelogStateBackendHandle {
+        static final String MATERIALIZED_FLAG = "materialized";
+        static final String NON_MATERIALIZED_FLAG = "nonMaterialized";
```

Comment:
nit: `private`?

Reply:
The implementation has been changed, no longger need these fields.